integrations/Integrations.md

```diff
@@ -27,7 +27,7 @@ openEquella has a number of ways to integrate with 3rd party systems
 # APIs
 
 ## APIs
-* REST API can be found at {{your Equella institution}}/apidocs.do
+* REST API can be found at `<your institution url>/apidocs.do`
```

Comment:
This change was mainly needed as the `{{text}}` is a format specifier for jekyll.

Reply:
I think it should specify that it's 'your openEQUELLA institution URL'

Reply:
Doneski.